openedx/features/course_experience/utils.py

```diff
@@ -136,6 +136,21 @@ def recurse_mark_scored(block):
             block['scored'] = False
             return False
 
+    def recurse_num_graded_problems(block):
```

Comment:
I'm surprised we don't have a more generic recurse tree + reduce, but makes sense to do this based on the surrounding methods